### DISCUSSION TITLE
Circular Rebalance Option Unavailable

### DISCUSSION THREAD

**Discussion by afxsoftware** [Ride-The-Lightning/RTL#1427] [C-Lightning]
I'm running Core Lightning on a Start9 Embassy. I have 2 channels open and would like to rebalance using RTL, but the option for circular rebalance isn't there. Please help.

**Comment by saubyk** [comment]
Hello @afxsoftware apologies for a late response, unfortunately circular rebalance is not supported by core lightning, hence it isn't available in the RTL UI

Given that you have only two channels open, I am curious about your objective to rebalance. What are you trying to achieve?
If you're trying to get inbound capacity, I would recommend trying out Liquidity Ads instead, which works on RTL

**Reply by afxsoftware** [reply]
I was a little confused because I'd see YouTube videos showing a circular rebalance option not realizing that RTL was using LND and not CLN. Thanks!
